src/main/java/uk/gov/justice/hmpps/prison/service/FinanceService.java

```diff
@@ -41,11 +48,21 @@
     private final AccountCodeRepository accountCodeRepository;
     private final OffenderSubAccountRepository offenderSubAccountRepository;
     private final OffenderTrustAccountRepository offenderTrustAccountRepository;
+    private final OffenderDamageObligationService offenderDamageObligationService;
 
     @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER", "GLOBAL_SEARCH"})
     public Account getBalances(final Long bookingId) {
+
+        var damageObligationBalance = bookingRepository.getLatestBookingByBookingId(bookingId)
```

Comment:
all I need is the offenderNo is using this method 'getLatestBookingByBookingId' a good way to get that value?

Reply:
done